### PR TITLE
postgres/sql: fix imports

### PR DESCRIPTION
**Pull request by gabor**
i adjusted the imports in the code, so that we do not import from core-grafana, only from `@grafana/*` packages. i only fixed the easy imports, some still remain that will need a more complex fix.

how to test:
- use the postgres datasource, create a new one, verify it works

### CODE REVIEW

File: public/app/plugins/datasource/postgres/configuration/useAutoDetectFeatures.ts [hunk 1]
```diff
@@ -38,7 +37,7 @@ export function useAutoDetectFeatures({ props, setVersionOptions }: Options) {
         // This is needed or else we get an error when we try to save the datasource.
         updateDatasourcePluginOption({ options, onOptionsChange }, 'version', result.datasource.version);
       } else {
-        const datasource = await getDatasourceSrv().loadDatasource(options.name);
+        const datasource = await getDataSourceSrv().get(options.name);
```
Comment (gabor) [inline public/app/plugins/datasource/postgres/configuration/useAutoDetectFeatures.ts:40]: the method `loadDatasource` is not exported on the data-type here, so i had to switch to `.get`. i checked the source code, and `.get()` will do some steps and then call `.loadDatasource`, so the change should be fine.

i also verified that below, when it calls `datasource.getVersion();`, the call is received by the postgres datasource plugin.

**Review by adamyeats — APPROVED** [review]
Nice @gabor! 🎉

**Review by aangelisc — APPROVED** [review]
This looks like a great start to me 👍

**Review by zoltanbedi — APPROVED** [review]
lgtm

File: public/app/features/plugins/sql/ResponseParser.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { uniqBy } from 'lodash';
 
 import { DataFrame, MetricFindValue } from '@grafana/data';
-import { ResponseParser as ResponseParserType } from 'app/features/plugins/sql/types';
+
+import { ResponseParser as ResponseParserType } from './types';
```
[the remark this answers is not in the document]
Comment (gabor) [reply public/app/features/plugins/sql/ResponseParser.ts]: done 👍

**Pull request merged**